save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/views/toprating/OrganizationRatingTab.kt

```diff
@@ -0,0 +1,151 @@
+@file:Suppress("HEADER_MISSING_IN_NON_SINGLE_CLASS_FILE")
+
+package com.saveourtool.save.frontend.components.views.toprating
+
+import com.saveourtool.save.entities.OrganizationDto
+import com.saveourtool.save.filters.OrganizationFilter
+import com.saveourtool.save.frontend.components.basic.nameFiltersRow
+import com.saveourtool.save.frontend.components.tables.*
+import com.saveourtool.save.frontend.externals.fontawesome.faTrophy
+import com.saveourtool.save.frontend.externals.fontawesome.fontAwesomeIcon
+import com.saveourtool.save.frontend.utils.*
+import com.saveourtool.save.frontend.utils.noopLoadingHandler
+import com.saveourtool.save.frontend.utils.noopResponseHandler
+import com.saveourtool.save.v1
+
+import js.core.jso
+import react.*
+import react.dom.html.ReactHTML.div
+import react.dom.html.ReactHTML.img
+import react.dom.html.ReactHTML.td
+import react.dom.html.ReactHTML.th
+import react.dom.html.ReactHTML.tr
+import react.router.dom.Link
+import web.cssom.*
+
+import kotlinx.browser.window
+import kotlinx.serialization.encodeToString
+import kotlinx.serialization.json.Json
+
+val renderOrganizationRatingTab: FC<Props> = FC { _ ->
```

Comment:
To my mind, `render` here is odd. Should be renamed to `organizationRatingTab`

Reply:
Ok. Renamed it :)